test/docs.jl

```diff
@@ -1551,3 +1551,7 @@ Base.@ccallable c51586_long()::Int = 3
 
 @test docstrings_equal(@doc(c51586_short()), doc"ensure we can document ccallable functions")
 @test docstrings_equal(@doc(c51586_long()), doc"ensure we can document ccallable functions")
+
+@testset "Docs docstrings" begin
+    @test isempty(Docs.undocumented_names(Docs))
```

Comment:
Once #52743 is merged, change to:
```suggestion
    undoc = Docs.undocumented_names(Docs)
    @test_broken isempty(Docs.undocumented_names(Docs))
    @test undoc == [Symbol("@var")]
```